SalesforceCommerceCloud/pwa-kit
# [App Extensibility ⚙️] Update generator to use static npm search result (@W-17273312@)

# Description



The PR updates the generator to use static results from npm search with the known packages.

The PR also bumps the extensibility preview release version for all packages in the monorepo.

- Core packages are released with the version `4.0.0-extensibility-preview.1`.
- Extension packages are released with the version `0.1.0-extensibility-preview.1`.

# Types of Changes




> Breaking changes include:
>
> - Removing a public function or component or prop
> - Adding a required argument to a function
> - Changing the data type of a function parameter or return value
> - Adding a new peer dependency to `package.json`

# Changes

- Update the generator to use static results from npm search with the known packages.
- Bumps the extensibility preview release version.

# How to Test-Drive This PR

- Run the generator to generate a new project using extensions`node packages/pwa-kit-create-app/scripts/create-mobify-app-dev.js`
- Start the generated project and verify things work as expected.

# Checklists




## General


## Accessibility Compliance

You must check off all items in **one** of the follow two lists:


_or..._


## Localization
